test/core/elem.wast

```diff
@@ -5,17 +5,29 @@
   (table $t 10 funcref)
   (func $f)
   (func $g)
-  (elem (table $t) (i32.const 0) func)
-  (elem (i32.const 0))
-  (elem (i32.const 0) $f $f)
-  (elem (offset (i32.const 0)))
-  (elem (offset (i32.const 0)) $f $f)
 
+  ;; Passive
+  (elem funcref)
+  (elem funcref (ref.func $f) (ref.func $f) (ref.null) (ref.func $g))
   (elem func)
-  (elem $s func)
   (elem func $f $f $g $g)
-  (elem $x func $f $f $g $g)
 
+  (elem $p1 funcref)
+  (elem $p2 funcref (ref.func $f) (ref.func $f) (ref.null) (ref.func $g))
+  (elem $p3 func)
+  (elem $p4 func $f $f $g $g)
+
+  ;; Active
+  (elem (table $t) (i32.const 0) funcref)
+  (elem (table $t) (i32.const 0) funcref (ref.func $f) (ref.null))
+  (elem (table $t) (i32.const 0) func)
+  (elem (table $t) (i32.const 0) func $f $g)
+  (elem (table $t) (i32.const 0) funcref)
+  (elem (table $t) (i32.const 0) funcref (ref.func $f) (ref.null))
+  (elem (table $t) (i32.const 0) func)
+  (elem (table $t) (i32.const 0) func $f $g)
```

Comment:
Aren't these just duplicates of the tests above?

Reply:
Oops, yes, removed (here and below).